xr_sample_project/Assets/SampleViewer/Samples/VRSample/Scripts/FadeScreen.cs

```diff
@@ -60,15 +61,21 @@ private IEnumerator FadeRoutine(float alphaIn, float alphaOut)
         {
             newColor = fadeColor;
             newColor.a = Mathf.Lerp(alphaIn, alphaOut, timer / fadeDuration);
-            rendererComponent.material.SetColor("_UnlitColor", newColor);
+            if(rendererComponent)
+            {
+                rendererComponent.material.SetColor("_UnlitColor", newColor);
+            }
             timer += Time.fixedDeltaTime;
             yield return null;
         }
 
         // To Confrim that the alpha finishes at the correct amount
         Color lastColor = fadeColor;
         lastColor.a = alphaOut;
-        rendererComponent.material.SetColor("_UnlitColor", lastColor);
+        if (rendererComponent)
+        {
+            rendererComponent.material.SetColor("_UnlitColor", lastColor);
```

Comment:
```suggestion
            rendererComponent?.material.SetColor("_UnlitColor", lastColor);
```
There is an easier way to check for `null` when calling methods.